src/reducers/settings.js

```diff
@@ -323,4 +323,15 @@ export const countervalueFirstSelector = (state: State) =>
 export const readOnlyModeEnabledSelector = (state: State) =>
   Platform.OS !== "android" && state.settings.readOnlyModeEnabled;
 
+// $FlowFixMe
+export const exportSettingsSelector = createSelector(
+  counterValueCurrencySelector,
+  state => state.settings.currenciesSettings,
+  (counterValueCurrency, currenciesSettings) => ({
+    counterValue: counterValueCurrency.ticker,
+    currenciesSettings,
+    developerModeEnabled: __DEV__,
```

Comment:
mmh, it's more the actual value of the settings developerModeEnabled. (this enable or not the testnet coins)

Reply:
also, the signature of that liveqr settings have recently changed https://github.com/LedgerHQ/ledger-live-common/blob/master/src/cross.js (in recent live-common version)
and it is already the case on desktop : https://github.com/LedgerHQ/ledger-live-desktop/blob/develop/src/reducers/settings.js#L248-L259

depending on which version of live-common we are on, we might want to HODL this PR for this new live-common version and adapt the present code.

Reply:
Couldn't understand how `pairExchanges` was stored on mobile, and for dev mode I couldn't find the switch in the settings reducer either. I'll look again tomorow :)

Reply:
Couldn't understand how `pairExchanges` was stored on mobile, and for dev mode I couldn't find the switch in the settings reducer either. I'll look again tomorow :)